lasagne/layers/merge.py

```diff
@@ -233,8 +233,7 @@ def autocrop_array_shapes(input_shapes, cropping):
             if cr is None:
                 result.append(sh)
             elif cr in {'lower', 'center', 'upper'}:
-                min_sh = None if any(x is None for x in sh) else min(sh)
-                result.append([min_sh] * len(sh))
+                result.append([min(sh)] * len(sh))
```

Comment:
This will break the tests, `None` needs to be handled explicitly since it cannot be compared to integers in Python 3.